irr/src/CMakeLists.txt

```diff
@@ -252,7 +246,11 @@ if(ENABLE_OPENGL OR ENABLE_OPENGL3)
 	find_package(OpenGL REQUIRED)
 endif()
 if(USE_SDL2)
-	find_package(SDL2 REQUIRED)
+	if(NOT ANDROID)
+		find_package(SDL2 REQUIRED)
+	else()
+		# provided by MinetestAndroidLibs.cmake
```

Comment:
I guess in the future we could package SDL2 in a way that the standard cmake way works...